contracts/src/PriceFeeds/MainnetPriceFeedBase.sol

```diff
@@ -11,7 +11,6 @@ import "../BorrowerOperations.sol";
 
 abstract contract MainnetPriceFeedBase is IPriceFeed, Ownable {
     // Dummy flag raised when the collateral branch gets shut down.
-    // Should be removed after actual shutdown logic is implemented.
```

Comment:
If the flag is here to stay (as it appears to be) then I would no longer call it a dummy.